config/manager/kustomization.yaml

```diff
@@ -1,8 +1,15 @@
 resources:
 - manager.yaml
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+- ../rbac
+
+
+# Adds namespace to all resources.
+namespace: multicluster-gateway-controller-system
+
 images:
 - name: controller
   newName: quay.io/kuadrant/multicluster-gateway-controller
   newTag: main
+
+patches:
+- path: manager_metrics_patch.yaml
```

Comment:
This should probably be reverted, it shouldn't include rbac here, or set a namespace.

manager_metrics_patch.yaml is a patch that should be called from outside of this kustomization  doing it here is making it redundant and you might as well just update manager.yaml